mobius/examples/crinkle_arc.rs

```diff
@@ -1,99 +1,109 @@
-use std::{
-    f64::consts::{PI, TAU},
-    io::Error,
-};
+use std::{f64::consts::PI, io::Error};
 
-use mobius::{algorithms::SemigroupIFS, geometry::ArcAngles, transformable::Cline};
 use mobius::{
-    cline_arc::ClineArc,
+    algorithms::SemigroupIFS,
+    geometry::{ArcAngles, DirectedEdge, GeneralizedCircle},
+    orthogonal_arcs::compute_orthogonal_circle,
+    svg_plot::union,
+    transformable::ClineArcTile,
+};
+use mobius::{
     geometry::{Circle, CircularArc},
     map_triple,
     rendering::Style,
     svg_plot::{render_views, style_geometry, View},
     Complex, Mobius,
 };
-use svg::node::element::Group;
-
-/// Compute an orthogonal circle through a and b.
-/// I've done this before for another project, see my explainer here:
-/// https://github.com/ptrgags/p5-sketchbook/tree/main/HyperbolicConnections#method-2-kite-analysis
-/// as well as the code
-/// https://github.com/ptrgags/p5-sketchbook/blob/458e47383ed8492cff3cc0bce4bded666b0672bc/HyperbolicConnections/boundaries.js#L35
-fn get_orthog_circle(circle: Circle, a: Complex, b: Complex) -> Circle {
-    let q = (a - b).mag();
-    let p = (4.0 / (4.0 - q * q)).sqrt();
-    let orthog_radius = 0.5 * p * q;
-
-    let angle_a = circle.get_angle(a).unwrap();
-    let angle_b = circle.get_angle(b).unwrap();
-
-    let angle_bisector = 0.5 * (angle_a + angle_b);
-    let angle_bisector = if (angle_b - angle_a) % TAU > PI {
-        (angle_bisector + PI) % TAU
-    } else {
-        angle_bisector
+
+fn compute_orthogonal_arc(arc: CircularArc, a: Complex, b: Complex) -> CircularArc {
```

Comment:
This should be moved to the new file and tested. Especially since large arcs seem to cause it to fall apart.